src/CMakeLists.txt

```diff
@@ -83,6 +83,93 @@ set(SOURCES
     utils/pdu_utils.cpp
 )
 
+set(HEADERS
+    ${LIBTINS_INCLUDE_DIR}tins/address_range.h
+    ${LIBTINS_INCLUDE_DIR}tins/arp.h
+    ${LIBTINS_INCLUDE_DIR}tins/bootp.h
+    ${LIBTINS_INCLUDE_DIR}tins/handshake_capturer.h
+    ${LIBTINS_INCLUDE_DIR}tins/stp.h
+    ${LIBTINS_INCLUDE_DIR}tins/pppoe.h
+    ${LIBTINS_INCLUDE_DIR}tins/config.h
+    ${LIBTINS_INCLUDE_DIR}tins/constants.h
+    ${LIBTINS_INCLUDE_DIR}tins/crypto.h
+    ${LIBTINS_INCLUDE_DIR}tins/cxxstd.h
+    ${LIBTINS_INCLUDE_DIR}tins/data_link_type.h
+    ${LIBTINS_INCLUDE_DIR}tins/detail/address_helpers.h
+    ${LIBTINS_INCLUDE_DIR}tins/detail/icmp_extension_helpers.h
+    ${LIBTINS_INCLUDE_DIR}tins/detail/pdu_helpers.h
+    ${LIBTINS_INCLUDE_DIR}tins/detail/sequence_number_helpers.h
+    ${LIBTINS_INCLUDE_DIR}tins/detail/smart_ptr.h
+    ${LIBTINS_INCLUDE_DIR}tins/detail/type_traits.h
+    ${LIBTINS_INCLUDE_DIR}tins/dhcp.h
+    ${LIBTINS_INCLUDE_DIR}tins/dhcpv6.h
+    ${LIBTINS_INCLUDE_DIR}tins/dns.h
+    ${LIBTINS_INCLUDE_DIR}tins/dot3.h
+    ${LIBTINS_INCLUDE_DIR}tins/dot1q.h
+    ${LIBTINS_INCLUDE_DIR}tins/dot11.h
+    ${LIBTINS_INCLUDE_DIR}tins/dot11/dot11_base.h
```

Comment:
I think dot11 related files should be split somewhere else as the whole protocol can be disabled so I don't think you should have it show up in your IDE if you explicitly disabled it.

Reply:
Hadn’t realised that - sounds sensible - will sort that.